### PR TITLE
Customizable multiselect printer

### PR DESCRIPTION
**Pull request by jochil**
### Description
Adding options to the `InteractiveMultiselectPrinter` to customize it:

- `NoFilter` to disable fuzzy search / filter
- `KeySelect` and `KeyConfirm` to use custom keys for selecting an item and confirm selection 

- [ ] Bug Fix
- [X] New Feature
- [X] Documentation
- [ ] Other

### Related Issue
Fixes #

### To-Do Checklist
- [X] I tested my changes
- [X] I have commented every method that I created/changed
- [X] I updated the examples to fit with my changes
- [ ] I have added tests for my newly created methods

### CODE REVIEW

**Comment by MarvinJWendt** [comment]
Thanks for the PR! I'll take a look soon 👍

File: interactive_multiselect_printer.go [hunk 1]
```diff
@@ -111,6 +123,10 @@ func (p *InteractiveMultiselectPrinter) Show(text ...string) ([]string, error) {
 		return nil, fmt.Errorf("could not start area: %w", err)
 	}
 
+	if !p.NoFilter && (p.KeyConfirm == keys.Space || p.KeySelect == keys.Space) {
+		return nil, fmt.Errorf("if filter/search is active, keys.Space can not use for KeySelect or KeyConfirm")
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:127]:
```suggestion
		return nil, fmt.Errorf("if filter/search is active, keys.Space can not be used for KeySelect or KeyConfirm")
```

File: interactive_multiselect_printer.go [hunk 2]
```diff
@@ -22,6 +22,8 @@ var (
 		MaxHeight:      5,
 		Selector:       ">",
 		SelectorStyle:  &ThemeDefault.SecondaryStyle,
+		KeySelect:      keys.Enter,
+		KeyConfirm:     keys.Tab,
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:26]: The methods `WithKeySelect` and `WithKeyConfirm` are missing 😉

File: interactive_multiselect_printer.go [hunk 3]
```diff
@@ -35,6 +37,7 @@ type InteractiveMultiselectPrinter struct {
 	MaxHeight      int
 	Selector       string
 	SelectorStyle  *Style
+	NoFilter       bool
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:40]: Could this variable name be changed to `Filter` and be true by default?

File: interactive_multiselect_printer.go [hunk 4]
```diff
@@ -70,6 +76,12 @@ func (p InteractiveMultiselectPrinter) WithMaxHeight(maxHeight int) *Interactive
 	return &p
 }
 
+// WithNoFilter sets the NoFilter option
+func (p InteractiveMultiselectPrinter) WithNoFilter() *InteractiveMultiselectPrinter {
```
Comment (MarvinJWendt) [inline interactive_multiselect_printer.go:80]: Could this be changed to `WithFilter`. Users could disable it with `WithFilter(false)` then.
In PTerm we usually just enable options by default with the `WithXxx` methods. For disabling, we use the approach above :)

**Review by MarvinJWendt — CHANGES_REQUESTED** [review]

**Comment by MarvinJWendt** [comment]
Hi @jochil, I have left some comments, the rest looks good! I'll merge it when you've fixed the comments 🚀

**Comment by jochil** [comment]
> Hi @jochil, I have left some comments, the rest looks good! I'll merge it when you've fixed the comments rocket

Thanks for the feedback... I fixed the comments

**Comment by jochil** [comment]
@MarvinJWendt Hi. Is there something else I can adjust, please let me know :)

**Comment by MarvinJWendt** [comment]
Hi @jochil, sorry for the long wait, I will review it now :)

**Review by MarvinJWendt — APPROVED** [review]
I just renamed the example. Thanks for the PR, and sorry for the delay. LGTM! 🚀

**Pull request merged**